### DISCUSSION TITLE
Consider reloading the config with every new plot

### DISCUSSION THREAD

**Discussion by aretheyalltaken2** [swar/Swar-Chia-Plot-Manager#277] [Ideas]
Sometimes (especially in the beginning) we may not know the right settings to perfectly optimise plotting.  In this case we may need to adjust the config.xaml file to tweak the settings.
In order to apply the new config we need to kill the plotting process, which if it is in flight may be part way through several plots.  

Perhaps if each time SwarManager starts a new plotting job it could reload the config, so then the most you would have to wait for to start your new settings is for the current plots to finish.

**Comment by raid5** [comment]
You can just restart the manager and it will use the new settings when the next plotting process is kicked off. You don't need to kill ongoing plotting unless there is something wrong with their setup.

`python manager.py restart`

**Reply by aretheyalltaken2** [reply]
Thank you for the tip - I didn't realise you could do that.
